gubernator/log_parser.py

```diff
@@ -107,17 +107,18 @@ def digest(data, objref_dict=None, filters=None, error_re=regex.error_re,
     if filters is None:
         filters = {'Namespace': '', 'UID': '', 'pod': '', 'ContainerID':''}
 
-    hilight_words = ["error", "fatal", "fail", "failed", "build timed out"]
+    highlight_words = ["error", "fatal", "fail", "failed", "build timed out",
+                       "undefined"]
```

Comment:
This only captures undefined golang build errors. I think the re I mentioned in the issue will be more helpful